docker-compose.yml

```diff
@@ -101,6 +99,16 @@ services:
       dockerfile: rust/Dockerfile
     volumes: *volumes
 
+  r:
+    # Usage:
+    #   docker-compose build r
```

Comment:
Sadly docker-compose doesn't support build dependencies, so We need to manually build the parent image `arrow:cpp`. Later a Makefile will resolve this inconvenience, but for now:

```
docker-compose build cpp
docker-compose build r
docker-compose run r
```

Reply:
ok no problem! Easy addition